src/pybrake/metrics.py

```diff
@@ -5,14 +5,12 @@
 from contextlib import contextmanager
 import time as pytime
 
+from . import constant
 from .utils import logger
 
 
 threadLocal = threading.local()
 
-FLUSH_PERIOD = 15
-_METRIC_KEY = "_ab_metric"
```

Comment:
I don't think it's a great idea to move all the constants into 1 file. It looks like we don't use this constant elsewhere, so why not keep it in this file? Same for other constants that are not re-used. Let's keep them in those files where they are used unless they're shared.

Reply:
Moved back all private variable which use in single file. Like:- _METRIC_KEY, _ERR_IP_RATE_LIMITED, _AB_URL_FORMAT.